docs/guide/snaps-rpc-api.md

```diff
@@ -226,6 +235,8 @@ interface WalletInstallSnapsResult {
 This method attempts to install the requested snaps, if they are permitted.
 If the installation of any snap fails, its object value on the result will contain an `error` property with the error that caused the installation to fail.
 
+Optionally you can specify a [SemVer range](https://www.npmjs.com/package/semver) of a Snap to be installed. Metamask will try to install a version that is max-satisfying of all the possible versions within that range. Should a Snap be already installed with a version that isn't in the requested range, Metamask will try to upgrade the installed Snap to satisfy the range, or an error returned if an upgrade is not possible or denied by the user.
```

Comment:
Is it our convention to capitalize "snap" or leave it uncapitalized?

Also Metamask -> MetaMask

Reply:
Also, "Metamask will try to install a version that is max-satisfying of all the possible versions within that range" sounds a bit strange. Maybe "MetaMask will try to install a version that satisfies the maximum version specified by the range?"

Reply:
Regarding the capitalization of "snap"/"snaps", we haven't established any conventions, but I've been doing this:
- "Snaps" refers to the feature / system as a whole.
- A "snap" / several "snaps" refers to individual snap(s) developed by someone.